garage/replay_buffer/her_replay_buffer.py

```diff
@@ -63,7 +60,7 @@ def _her_sample_transitions(episode_batch, sample_batch_size):
             for k, rk in zip(["next_achieved_goal", "goal"],
                              list(reward_params_keys)[:-1])
         }
-        reward_params["info"] = info
+        reward_params["info"] = {}
```

Comment:
I noticed that info is not included in HER replay buffer. I think this is useful for reward computation for some cases.

Reply:
What's the reason for this change? It doesn't look correct to me.

Reply:
For now all the GoalEnv.compute_reward in gym doesn't use information in `info`, though it needs this parameter. 

The reason I don't store `info` in HER replay buffer is, when construct replay buffer, we need to get all the keys and value shapes. To get all the keys in info, I didn't find other way but call
```
env.reset()
_, _, _, info = env.step(env.action_space.sample())
```
But is it proper to call `env.reset()` in the constructor of replay buffer?